src/main/java/club/sk1er/mods/autogg/command/AutoGGCommand.java

```diff
@@ -1,127 +1,33 @@
-    public void refresh() {
-        ModCoreAPI.getMinecraftUtil().sendMessage(prefix, ChatColor.YELLOW + "Fetching triggers...");
-        AutoGG.downloadTriggers(true);
-        AutoGGListener.switchTriggerset();
-    }
-
-    @SubCommand(value = "triggers", description = "Dump the currently fetched triggers for debugging.")
-    public void triggers() {
-        for (String key : AutoGG.ggRegexes.keySet()) {
-            if (!AutoGG.ggRegexes.get(key).isEmpty()) {
-                ModCoreAPI.getMinecraftUtil().sendMessage(prefix, ChatColor.AQUA +
-                    key.replaceAll("_", " ").toUpperCase(Locale.ENGLISH) + ":");
-                for (Pattern pattern : AutoGG.ggRegexes.get(key)) {
-                    ModCoreAPI.getMinecraftUtil().sendMessage("  ", pattern.toString());
-                }
-            }
-        }
-
-        for (String key : AutoGG.otherRegexes.keySet()) {
-            if (!"$^".equals(AutoGG.otherRegexes.get(key).toString())) {
-                ModCoreAPI.getMinecraftUtil().sendMessage(prefix, ChatColor.AQUA +
-                    key.replaceAll("_", " ").toUpperCase(Locale.ENGLISH) + ": " + ChatColor.RESET +
-                    AutoGG.otherRegexes.get(key));
-            }
-        }
-    }
-
-    @SubCommand(value = "info", description = "Display information about AutoGG.")
-    public void info() {
-        ModCoreAPI.getMinecraftUtil().sendMessage(prefix, ChatColor.GREEN + "Mod Version: " + AutoGG.VERSION);
-        try {
-            int triggersSize = AutoGG.ggRegexes.get("triggers").size();
-            int casualTriggersSize = AutoGG.ggRegexes.get("casual_triggers").size();
-            ModCoreAPI.getMinecraftUtil().sendMessage(prefix, ChatColor.GREEN +
-                "Triggers Version: " +
-                AutoGG.triggerMeta.get("version").replaceAll("\"", ""));
-            ModCoreAPI.getMinecraftUtil().sendMessage(prefix, ChatColor.GREEN +
-                "Triggers last updated on " +
-                LOCALE_FORMAT.format(parseDate(AutoGG.triggerMeta.get("upload_date")
-                    .replaceAll("\"", ""))));
-            ModCoreAPI.getMinecraftUtil().sendMessage(prefix, ChatColor.GREEN +
-                "Triggers info message: " +
-                AutoGG.triggerMeta.get("note").replaceAll("\"", ""));
-            ModCoreAPI.getMinecraftUtil().sendMessage(prefix, ChatColor.GREEN +
-                Integer.toString(triggersSize) + " Trigger" + (triggersSize == 1 ? "" : "s") + ", " + casualTriggersSize + " Casual Trigger" +
-                (casualTriggersSize == 1 ? "" : "s"));
-        } catch (NullPointerException e) {
-            ModCoreAPI.getMinecraftUtil().sendMessage(prefix, ChatColor.RED +
-                "Could not get Trigger Meta! Were the triggers downloaded properly?");
-            AutoGG.instance.getLogger().error("Could not get trigger meta.", e);
+        GuiScreen gui = AutoGG.INSTANCE.getAutoGGConfig().gui();
```

Comment:
can be simplified to `GuiUtil.open(Objects.requireNonNull(AutoGG.INSTANCE.getAutoGGConfig().gui()))`